go.mod

```diff
@@ -38,6 +38,7 @@ require (
 	github.com/fatih/color v1.15.0
 	github.com/fsnotify/fsnotify v1.7.0
 	github.com/getkin/kin-openapi v0.126.0
+	github.com/go-git/go-billy/v5 v5.5.0
```

Comment:
Looks like an unrelated change to `go.mod`? Try running `go mod tidy`

Reply:
This line is still there? It's not referenced in this PR, so seems like a mistake?